playground/TestShop/MyFrontend/Program.cs

```diff
@@ -11,7 +11,7 @@
 builder.Services.AddHttpClient<CatalogServiceClient>(c => c.BaseAddress = new("https+http://catalogservice"));
 
 builder.Services.AddSingleton<BasketServiceClient>()
-                .AddGrpcClient<Basket.BasketClient>(o => o.Address = new("http://basketservice"));
+                .AddGrpcClient<Basket.BasketClient>(o => o.Address = new("https+http://basketservice"));
```

Comment:
This breaks the gRPC client

Reply:
Ah right! We'll need to figure something out on that front, but in the meantime I'll fix the TestShop.